.gitignore

```
@@ -14,5 +14,15 @@ dist
 reports
 coverage
 
+# .env
+.eslintcache
 .env
+.venv
+__pycache__/
+/public/assets
+/public/locales
+/public/node_modules
+/public/pdf.worker.min.mjs
 .eslintcache
+.env
+.DS_Store
```

This looks like some leftovers from development?
I don't see why we would have python stuff in our repo.
not sure why the pdf worker is in here if it's still committed as the .js 🤔 is that on purpose?
```suggestion
.env
.eslintcache
```

---

As for this change here, you are correct on your points. But I added the pdf worker here in an attempt to deal with the failing in loading PDFs, as there were two versions of the library. I still have not come to tangible fixes on this end.